proposals/incubation/cert-manager.md

```diff
@@ -0,0 +1,138 @@
+# cert-manager Incubation Proposal
+
+cert-manager joined the CNCF as a sandbox project just over a year ago, and we firmly believe that the project is ready to take the next step and graduate to the Incubation stage.
+
+cert-manager provides reliable, automated Kubernetes-native management of X.509 certificates. It reduces the risk of certificate-related outages and enables PKI at scale, and is treated by many teams as de-facto part of the core components of a Kubernetes or OpenShift deployment.
+
+The project comprises cert-manager itself - which adds Kubernetes Custom Resource Definitions (CRDs) for certificate-related concepts, and provides controllers to manage them - along with a selection of subprojects which focus on solving other issues relating to certificate management.
```

Comment:
> long with a selection of subprojects which focus on solving other issues relating to certificate management

This makes it a little unclear whether it'd be just cert-manager/cert-manager that'd be graduating or the other subprojects too

Reply:
This is a thought-provoking comment. For sure the main thing here is cert-manager itself, but is it "cert-manager the product" or "cert-manager the project" that's being proposed for incubation?

E.g. [Argo's CNCF project page](https://www.cncf.io/projects/argo/) has a GitHub link directly to [the main workflows repo](https://github.com/argoproj/argo-workflows) but the argo website lists all of the sub projects (e.g. argo-cd, events, etc) in the title bar. Is argo-cd an incubating project? If they started a new greenfield project called "argo-certs" which managed TLS certs, would that qualify as incubating? I don't know how the CNCF draws those lines.

Maybe something like this? :thinking: 

```suggestion
The project comprises cert-manager itself - which adds Kubernetes Custom Resource Definitions (CRDs) for certificate-related concepts, and provides controllers to manage them - along with a selection of subprojects which focus on solving other issues relating to certificate management.

We'll primarily discuss cert-manager in this proposal, although we do intend to keep developing the other projects, all of which tie into cert-manager.
```

Reply:
Oh, I'd missed that part in argo's proposal, thanks for linking. I don't think any of the subprojects would pass the incubation criteria individually, and so I lean towards (b) in what you described:

>  b) there are associated tools, but we don't propose those to incubation, please only consider cert-manager repo/tool

I'll make that change :+1: